articles/quickstart/spa/angular-next/02-calling-an-api.md

```diff
@@ -0,0 +1,186 @@
+
+The following section demonstrates how to configure the application module to call APIs.
+
+### Configure the Auth Module
+
+The `AuthModule` setup is similar to the one discussed in the [Configure the `Auth0Provider` component](/quickstart/spa/angular-next##register-and-configure-the-authentication-module) section: you import the `AuthModule` type from the SDK into your Angular application module and call the static `forRoot` method, passing the `domain` and `clientId` properties. The values of these two props come from the ["Settings" values](https://auth0.com/docs/quickstart/spa/angular-next#configure-auth0) of the single-page application you've registered with Auth0.
+
+However, your Angular application needs to pass an access token when it calls a target API to access private resources. You can [request an access token](https://auth0.com/docs/tokens/guides/get-access-tokens) in a format that the API can verify by passing the `audience` and `scope` props to `AuthModule.forRoot`.
+
+In addition, the configuration should specify which requests should have an `Authorization` header added to them with the access token.
+
+The following example shows how to set up your Angular application module, with all the necessary imported types and the configuration to call the management API:
+
+```javascript
+// Import the injector module and the Angular types you'll need
+import { HttpClientModule, HTTP_INTERCEPTORS } from '@angular/common/http';
+import { AuthHttpInterceptor } from '@auth0/auth0-angular';
+
+@NgModule({
+  declarations: [],
+  imports: [
+    BrowserModule,
+
+    // Register HttpClient for making API calls
+    HttpClientModule,
+    
+    AppRoutingModule,
+    AuthModule.forRoot({
+      domain: "${account.namespace}",
+      clientId: "${account.clientId}",
+
+      // Specify the audience for the management API, allowing us to make API calls
+      audience: "https://${account.namespace}/api/v2/",
+
+      // Specify the necessary scopes for getting user information
+      scope: "read:current_user",
+
+      // Specify configuration for the interceptor              
+      httpInterceptor: {
+        allowedList: [
+          {
+            // Match any request that starts 'https://${account.namespace}/api/v2/' (note the asterisk)
+            uri: 'https://${account.namespace}/api/v2/*'
+            tokenOptions: {
+              // The attached token should target this audience
+              audience: 'https://${account.namespace}/api/v2/',
+
+              // The attached token should have these scopes
+              scope: 'read:current_user'
+            }
+          }
+        ]
+      }
+    }),
+  ],
+  providers: [
+    // Register the `AuthHttpInterceptor` to automatically attach the 'Authorization' header
+    { provide: HTTP_INTERCEPTORS, useClass: AuthHttpInterceptor, multi: true },
+  ],
+  bootstrap: [AppComponent],
+})
+```
+
```

Comment:
if there's any way of referencing the section of the preceding chapter where the other config essential entries are explained, would be good to do so- or at least to mention that those explanations are there, and here we focus only on the new, API related stuff